libcxx/test/libcxx/utilities/expected/expected.void/transform_error.mandates.verify.cpp

```diff
@@ -10,11 +10,12 @@
 // minumum required version these obsolete tests can be removed.
 // TODO(LLVM-20) remove spurious clang diagnostic tests.
 
-// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
-
 // With clang-cl, some warnings have a 'which is a Microsoft extension' suffix
-// which break the tests.
-// XFAIL: msvc
+// which break the tests. But #102851 will turn it into an error, making the test pass.
+// However, upstream libcxx buildbots do not build clang from source while testing, so
+// this tests still expected to fail on these bots. Mark msvc as unsupported to mitigate
+// this issue.
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, msvc
```

Comment:
We should probably split the msvc bit out into its own `UNSUPPORTED:` group w/ this comment and a tracking issue on github, since once libcxx CI builds with a new enough clang, this could go back to being supported. Same for the other test.

Reply:
I have moved it into a new unsupported group.
For filing an upstream tracking issue, what content should I put into it? We can file a bug to ask rolling the clang in libcxx bots, but libcxx bots uses the latest LLVM release so they will be rolled once llvm has a new release. It would be great that all libcxx bots build clang from ToT but I am not sure if that fits libcxx maintainer's best interests.